core/templates/dev/head/pages/exploration_player/TutorCardDirective.js

```diff
@@ -148,6 +173,79 @@ oppia.directive('tutorCard', [
             });
           };
 
+          $scope.playPauseAudioTranslation = function() {
+            // TODO(tjiang11): Change from on-demand loading to pre-loading.
+
+            if (cardIndexOfPlayingAudio === undefined) {
+              cardIndexOfPlayingAudio =
+                playerPositionService.getActiveCardIndex();
+
+              // TODO(tjiang11): On first play, ask learner to pick language
+              // and subsequently for confirmation to use bandwidth 
+              // to download audio files.
+            }
+
+            $scope.showExtraAudioControls = true;
+
+            if (!AudioPlayerService.isPlaying()) {
+              if (AudioPlayerService.trackLoaded() && isSameAudio()) {
+                AudioPlayerService.play();
+              } else {
+                loadAndPlayAudioTranslation();
+              }
+            } else {
+              AudioPlayerService.pause();
+              if (!isSameAudio()) {
+                loadAndPlayAudioTranslation();
```

Comment:
Hm, that doesn't sound like "pause" to me. What if we just cut the audio once you leave the card? It seems odd to even have the audio for one card playing while you're looking at another card; it doesn't match up anymore and there's no obvious indicator of that.